packages/@aws-cdk-testing/framework-integ/test/aws-ecs-patterns/test/ec2/integ.alb-ecs-service-command-entry-point.ts

```diff
@@ -4,8 +4,9 @@ import * as ecs from 'aws-cdk-lib/aws-ecs';
 import * as cdk from 'aws-cdk-lib';
 import * as integ from '@aws-cdk/integ-tests-alpha';
 import * as ecsPatterns from 'aws-cdk-lib/aws-ecs-patterns';
+import { AUTOSCALING_DISABLE_LAUNCH_CONFIG } from 'aws-cdk-lib/cx-api';
 
-const app = new cdk.App();
+const app = new cdk.App({ postCliContext: { [AUTOSCALING_DISABLE_LAUNCH_CONFIG]: false } });
```

Comment:
This integ test is unsuccessful as it is, i.e., without any changes. The behavior is the same regardless of whether a launch configuration or a launch template is used. I'm continuing to investigate, but didn't want this to be a blocker since the issue is unrelated to the changes in this PR.